### ISSUE TITLE
Classweighting of SegNet?

### ISSUE THREAD

**Issue by qiaobaiyisheng** [meetshah1995/pytorch-semseg#202]
If I use the SenNet, do I need to compute the classweighting mentioned in the paper?

**Comment by Qirui-Y** [comment]
Hello, are you solved problem ? Can you share your config file?

**Comment by qiaobaiyisheng** [comment]
`model:
    arch: segnet
data:
    dataset: camvid
    train_split: train
    val_split: val
    img_rows: 360
    img_cols: 480
    path: /your/path/pytorch-semseg-master/data/CamVid/
training:
    train_iters: 150000
    batch_size: 8
    val_interval: 100
    n_workers: 16
    print_interval: 50
    optimizer:
        name: 'sgd'
        lr: 1.0e-10
        weight_decay: 0.0005
        momentum: 0.99
    loss:
        name: 'cross_entropy'
        size_average: False
    lr_schedule:
    resume: segnet_camvid_model.pkl
`

I didn't found where to compute the classweighting in this code, so directly run the program with the config file above. The result on CamVid is about 0.58(mIOU), which is close to the experimental result in the proposed  SegNet paper.

**Comment by Qirui-Y** [comment]
OK，thank you very much,I am try it!
